runtime/jcl/common/jclcinit.c

```diff
@@ -68,86 +68,60 @@ jint computeFullVersionString(J9JavaVM* vm)
 	PORT_ACCESS_FROM_JAVAVM(vm);
 	const char *osarch;
 	const char *osname;
-	const char *vmVersion = NULL;
-	const char *gcVersion = NULL;
 	char fullversion[512];
 	char vminfo[512];
-#ifdef J9VM_INTERP_NATIVE_SUPPORT
-	J9JITConfig *jitConfig;
-#endif
+	char j2se_version_info[10];
+
 #if defined(J9VM_INTERP_NATIVE_SUPPORT)
+	J9JITConfig *jitConfig;
+
 	UDATA jitEnabled = 0;
 	UDATA aotEnabled = 0;
 #endif
 
-	strcpy(fullversion, "JRE ");
-	strcpy(vminfo, "JRE ");
 	switch(J2SE_VERSION(vm) & J2SE_VERSION_MASK) {
 	case J2SE_17:
 		if ((J2SE_VERSION(vm) & J2SE_RELEASE_MASK) == J2SE_17) {
-			strcat(fullversion, "1.7.0 ");
-			strcat(vminfo, "1.7.0 ");
+			strcpy(j2se_version_info, "1.7.0");
```

Comment:
If we change the type of `j2se_version_info` to `const char *` we can just use a pointer assignment:
```
j2se_version_info = "1.7.0";
```
Likewise with the other targets of `strcpy`.